mappings/net/minecraft/server/world/ServerWorld.mapping

```diff
@@ -45,6 +45,9 @@ CLASS ve net/minecraft/server/world/ServerWorld
 	METHOD a tickChunk (Lbww;I)V
 		ARG 1 chunk
 		ARG 2 randomTickSpeed
+	METHOD a isInterestingPointAt (Lev;I)Z
```

Comment:
Are interesting points the same as points of interest? If yes, I think point of interest is better, to be consistent with the other names.

Reply:
There is another integer parameter in the end which indicates the interest level (for a village), with 1 being the village center and 5 being no village. The interest level is associated with chunk sections. The `isVillageAt(BlockPos)` literally calls `isInterestingPointAt(BlockPos, 1)`